traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
+
+      {% if not user.is_anonymous %}
+      <div class="post-comment__container">
+        <div class="post-comment__header">
+          <span uk-icon="user"></span>
+          <span>Comment as {{user.get_full_name}}</span>
+          <button class="uk-button button button--primary post-comment__submit" onclick="commentButtonClicked()">Submit</button>
+        </div>
+
+        <div class="post-comment__body">
+          <textarea 
+          id="commentBody" 
+          class="uk-textarea post-comment__textarea" 
+          placeholder="Leave a commment..."></textarea>
+        </div>
+      </div>
+      {% endif%}
+
+      {% for event in history_events|dictsortreversed:"date" %}
+      {% if event.type == 'comment' %}
+      <div class="event event--comment">
+        <span class="event__title">{{event.content.author.get_full_name}} commented</span>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+        <p>{{event.content.body}}</p>
+      </div>
+
+      {% elif event.type == 'review' %}
+      <div class="event">
+        <p class="event__title"> 
+          {{event.content.reviewer.get_full_name}} reviewed mapping of trait "{{trait.name}}"
+          to term "{{event.content.mapping_id.term_id.curie}} - {{event.content.mapping_id.term_id.label}}"
+        </p>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+      </div>
```

Comment:
For the review entries:
![image](https://user-images.githubusercontent.com/10669118/89111475-2adebf00-d44e-11ea-8716-96f40642f547.png)

This can be changed to:

**Kirill Tsukanov** confirmed the mapping on **August 1st 2020, 23:08**  
**[Orphanet:586](http://www.orpha.net/ORDO/Orphanet_586) — Cystic fibrosis**